bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -356,7 +356,7 @@ The channel _command_ provides options
 - _vehicle-finder_
 - _door-lock_
 - _door-unlock_
-- _horn-low_
+- _horn-blow_
```

Comment:
Looking very quickly at the code, I think there is another mismatch for another command: this is not "flash-lights" but "light-flash". Can you please fix that in README, it is only 4 lines above your current change.

Reply:
Right, it must be "light-flash" not "flash-lights".
I fixed the README.md